package.json

```diff
@@ -104,7 +104,7 @@
     "ember-bunsen-core": "0.24.0",
     "ember-cli-babel": "^5.1.8",
     "ember-cli-htmlbars": "^1.0.3",
-    "ember-frost-core": "1.12.0",
+    "ember-frost-core": "1.13.0",
```

Comment:
Seems wrong to have a pinned version in `dependencies` too. Is it true that this addon will break if the consuming app is using a later version? If not, it should probably allow a more recent, version-compatible code to be installed. Otherwise you'll end up with two copies in `node_modules` and I'm not sure which one lands in the app.